docs/products/mysql/howto/prevent-disk-full.rst

```diff
@@ -0,0 +1,45 @@
+Prevent MySQL disk full
+=======================
+
+In the event that your MySQL server runs out of disk space, the service will start malfunctioning. Running out of disk space will also prevent backups from being properly created.
+
+To prevent this from happening, Aiven will automatically detect when your service is running out of free space, and prevent further writes to it. This process is done by setting the MySQL ``@@GLOBAL.read_only`` flag to ``1``. The threshold for moving to this state is when your disk usage is at 97% or higher.
+
+Once your service has been made ``read-only``, the service will report errors when you attempt to insert, update, or delete data::
+
+    ERROR 1290 (HY000): The MySQL server is running with the --read-only option so it cannot execute this statement  
+
+
+Actions to make more space available
+------------------------------------
+
+Identify and optimize problem tables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+InnoDB does not reclaim unused disk space by default and this can cause a disk to become full. 
+
+Read the help article MySQL disk usage for more information.
```

Comment:
Missing link for `MySQL disk usage`

Presumably still https://help.aiven.io/en/articles/4808068-mysql-disk-usage, unless it's been ported.

Reply:
thanks, fixed!